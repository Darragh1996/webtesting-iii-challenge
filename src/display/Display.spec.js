// Test away!
import React from "react";
import * as rtl from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Display from './Display';
import { exportAllDeclaration } from "@babel/types";

afterEach(rtl.cleanup);

let wrapper;
beforeEach(() => {
    wrapper = rtl.render(<Display />)
});

it('renders without crashing',() => {
    expect(wrapper.container).toMatchSnapshot();
});

it('renders elements properly -> DEFUALT PROPS', () => {
    
})

