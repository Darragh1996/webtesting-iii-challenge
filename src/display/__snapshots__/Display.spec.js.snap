// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements properly -> CLOSED SET TO TRUE 1`] = `
<div>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
    >
      Unlocked
    </div>
    <div
      class="led red-led"
    >
      Closed
    </div>
  </div>
</div>
`;

exports[`renders elements properly -> DEFUALT PROPS 1`] = `
<div>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
    >
      Unlocked
    </div>
    <div
      class="led green-led"
    >
      Open
    </div>
  </div>
</div>
`;

exports[`renders without crashing 1`] = `
<div>
  <div
    class="display panel"
  >
    <div
      class="led green-led"
    >
      Unlocked
    </div>
    <div
      class="led green-led"
    >
      Open
    </div>
  </div>
</div>
`;
